packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -26,12 +26,14 @@
 import {
   LONG_TASK,
   LARGEST_CONTENTFUL_PAINT,
-  FIRST_CONTENTFUL_PAINT
+  FIRST_CONTENTFUL_PAINT,
+  FIRST_INPUT
 } from '../common/constants'
 import { noop, PERF } from '../common/utils'
-import { metrics } from '../common/metrics'
 import Span from './span'
 
+export const metrics = {}
```

Comment:
need pre initialisation of FCP here. 
```js
export const metrics = {
   fcp: 0
}
```

We use this to check TBT https://github.com/elastic/apm-agent-rum-js/blob/cac9bfc0bbd14ad52e998ae95c537591765929b9/packages/rum-core/src/performance-monitoring/perf-entry-recorder.js#L111